lesson_15/class_rhombus.py

```diff
@@ -0,0 +1,35 @@
+class Rhombus:
+
+    side_a: int
+    angle_a: int
+
+    def __init__(self, side_a: int, angle_a: int):
+        self.side_a = side_a
+        self.angle_a = angle_a
+
+    def rhombus_side_a(self):
+        return f"Side a = {self.side_a}"
+
+    def rhombus_angle_a(self):
+        return f"Angle a = {self.angle_a}"
+
+    def __setattr__(self, name, value):
+        if name == 'side_a':
+            if value < 0:
```

Comment:
It is better to split this If statements into separate private methods. And only call them in __setattr__ method here.